forms/src/main/java/org/odk/collect/forms/FormListItem.kt

```diff
@@ -0,0 +1,10 @@
+package org.odk.collect.forms
+
+data class FormListItem(
+    val downloadURL: String,
+    val formID: String,
+    val version: String,
+    val hashWithPrefix: String?,
```

Comment:
It'd be good to think about if any of the other fields should `String?` here. If so, we missing test coverage of whatever case that would be from.